### PR TITLE
RHDEVDOCS-2671 - Logging support considerations 

### PR DESCRIPTION
**Pull request by libander**
Version(s): 4.10+

Issue: [RHDEVDOCS-2671](https://issues.redhat.com//browse/RHDEVDOCS-2671) 

Link to docs preview(s): https://drive.google.com/drive/folders/1_KHeuTVpusOnWV-PfTbDjVadFXr_t6ME?usp=sharing 

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/logging-support-considerations.adoc [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+* *Deploying logging to namespaces not specified in the documentation.*
+* *Installing custom Elasticsearch, Kibana, Fluentd, or Loki instances on {product-title}.*
+* *Changes to the Kibana Custom Resource (CR) or Elasticsearch CR.*
+* *Changes to secrets or configmaps not specified in the documentation.*
+
+The {logging-title} is:
+* For forwarding logs to various supported systems.
+* An opinionated collector and normalizer of application, infrastructure and audit logs.
+* Designed for optional use with low retention managed centralized storage.
```
Comment (jcantrill) [inline modules/logging-support-considerations.adoc:22]: everything lgtm  other then I do not understand what this bullet means

**Comment by libander** [comment]
@r2d2rnd - Any feedback?

**Comment by r2d2rnd** [comment]
Many thanks for this. It looks good to me

**Review by anpingli — COMMENTED** [review]
/lgtm

**Review by shipsing — CHANGES_REQUESTED** [review]
@libander Please see my suggestions. Thanks

File: modules/logging-support-considerations.adoc [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+include::snippets/logging-compatibility-snip.adoc[]
+
+include::snippets/logging-supported-config-snip.adoc[]
+
+The following modifications are explicitly not supported:
+
+* *Deploying logging to namespaces not specified in the documentation.*
+* *Installing custom Elasticsearch, Kibana, Fluentd, or Loki instances on {product-title}.*
+* *Changes to the Kibana Custom Resource (CR) or Elasticsearch CR.*
+* *Changes to secrets or configmaps not specified in the documentation.*
```
[the remark this answers is not in the document]
Comment (libander) [reply modules/logging-support-considerations.adoc:17]: > Why did we bold the text? Is there any recommendation to highlight this piece of information? 

https://docs.openshift.com/container-platform/4.12/monitoring/configuring-the-monitoring-stack.html#support-considerations_configuring-the-monitoring-stack - was used as a reference per the request of Support Engineering.

File: modules/logging-support-considerations.adoc [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+The {logging-title} is:
+
+* For forwarding logs to various supported systems.
+* An opinionated collector and normalizer of application, infrastructure, and audit logs.
```
[the remark this answers is not in the document]
Comment (libander) [reply modules/logging-support-considerations.adoc:22]: Opinionated has a well understood meaning in the context of software and is not prohibited by the IBM Style Guide, Red Hat Supplementary Style Guide, or the OpenShift Contribution Guide glossary.

**Pull request merged**